### PR TITLE
Small cleanup

### PR DESCRIPTION
**Pull request by dek3rr**
-cleaner mass conversions and wheel speed estimation in gm and ford
-fixed some comment typos
-clearer IF conditions in honda/carcontroller

### CODE REVIEW

**Review by rbiasini — COMMENTED** [review]
Looks good

File: selfdrive/car/honda/carcontroller.py [hunk 1]
```diff
@@ -90,13 +90,11 @@ def update(self, sendcan, enabled, CS, frame, actuators, \
...
       hud_lanes = 0
 
-    # TODO: factor this out better
-    if enabled:
-      if hud_show_car:
-        hud_car = 2
-      else:
-        hud_car = 1
-    else:
+    if enabled and hud_show_car:
```
[the remark this answers is not in the document]
Comment (dekerr) [reply selfdrive/car/honda/carcontroller.py]: you're right. that does look sloppy

[the remark this answers is not in the document]
Comment (dekerr) [reply selfdrive/car/honda/carcontroller.py]: if not enabled:
        hud_car = 0
    elif not hud_show_car:
        hud_car = 1 	
    else: 
        hud_car = 2
How do you feel about something like this?

[the remark this answers is not in the document]
Comment (rbiasini) [reply selfdrive/car/honda/carcontroller.py]: still prefer the original :)

[the remark this answers is not in the document]
Comment (dekerr) [reply selfdrive/car/honda/carcontroller.py]: haha you win

**Pull request merged**